src/gsi/radinfo.f90

```diff
@@ -1135,7 +1135,7 @@ subroutine radinfo_read
                 nusis(j),nuchan(j),' not found in satbias_in file - set to zero '
              endif
           end do
-140       format(i4,1x,a20,12f12.6)
+140       format(i5,1x,a20,12f12.6)
```

Comment:
This is a nitpick comment.  Feel free to ignore if you prefer.

Line 808 changes `i4` to `i6` for `j`.  Line 1138 changes `i4` to `i5` for `j`.  Why the difference?  

The first `j` counts the entries in the satinfo file.  The second `j` ranges from 1 to `jpch_rad`.  Variable `jpch_rad` is set to the first `j`.  Given this, it seems should use the same integer field width in the format statements on lines 808 and 1138.

Reply:
@RussTreadon-NOAA I have a GEOXO meeting today and tomorrow.  Not sure if I will get to this tonight.   I would also like to try removing nrec from the read_* and combine_radobs routines.  This would push everything into next week.  Is this acceptable?

Reply:
Whatever works for you, works for me @wx20jjung.  Take the time you need to get this PR into the state you want.

Reply:
I made the write statements in radinfo.f90 consistent.  I found 2 instances of initializing nrec in read_gmi.f90.  I removed the first one.